pkg/antctl/raw/multicluster/destroy.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package multicluster
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"antrea.io/antrea/pkg/antctl/raw/multicluster/common"
+)
+
+type destroyOptions struct {
+	namespace  string
+	clusterSet string
+}
+
+var destroyOpts *destroyOptions
+
+func (o *destroyOptions) validate() error {
+	if o.namespace == "" {
+		return fmt.Errorf("Namespace is required")
+	}
+	if o.clusterSet == "" {
+		return fmt.Errorf("ClusterSet is required")
+	}
+
+	return nil
+}
+
+var destroyExamples = strings.Trim(`
+# Destroy ClusterSet in the leader cluster
+  antctl mc destroy --namespace antrea-multicluster --clusterset test-clusterset
```

Comment:
I feel it's better to say 'clean', or 'wipe'? @jianjuns  any suggestion?

Reply:
As we use "init" for setup, I think we should use either "destroy" or "delete" for removal.